RecoLocalTracker/SubCollectionProducers/src/ClusterMultiplicityFilter.cc

```diff
@@ -15,19 +15,15 @@
 #include <memory>
 
 #include "RecoLocalTracker/SubCollectionProducers/interface/ClusterMultiplicityFilter.h"
-
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
-
-#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
-#include "DataFormats/Common/interface/DetSetVector.h"
-
 #include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
 
 
-ClusterMultiplicityFilter::ClusterMultiplicityFilter(const edm::ParameterSet& iConfig)
+ClusterMultiplicityFilter::ClusterMultiplicityFilter(const edm::ParameterSet& iConfig) :
+  maxNumberOfClusters_(iConfig.getUntrackedParameter<unsigned int>("MaxNumberOfClusters")),
```

Comment:
both config parameters affect the result of the code, so, they should be made tracked (remove "Untracked") 
[not sure if this would trigger changes in large number of places: let's discuss if it does, given that list]